app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```

Comment:
This line adds the `Person` instance to the `people` dictionary using the person's name as the key. Ensure that names are unique to avoid overwriting existing entries.